cmake/developer_package/compile_flags/os_flags.cmake

```diff
@@ -318,6 +318,18 @@ elseif(CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG)
     ie_add_compiler_flags(-fsigned-char)
 endif()
 
+file(RELATIVE_PATH OV_RELATIVE_BIN_PATH ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR})
```

Comment:
does it imply that binary folder is located inside the source dir?  If no, what is logic here?